Docs/myisam.txt

```diff
@@ -60,9 +60,9 @@ See MySQL Appendix B "Choosing a table type".
 MyISAM allows about 32 indexes. However the official MySQL limit is 16 until MySQL 4.0.
```

Comment:
The content of this document is extremely dated, referring to MySQL 3.23.5 as the current release etc. so perhaps @vuvova can comment on whether this document is still needed, or should, separately to this PR, be updated at some point.